tests/playwright/django-ory-cloud.spec.ts

```diff
@@ -0,0 +1,77 @@
+import { test, expect, Page } from "@playwright/test"
+import { randomEmail, randomString } from "./helpers"
+
+const login = async (page: Page) => {
+  const email = randomEmail()
+  await page.fill('[name="traits.email"]', email)
+  await page.fill('[name="password"]', randomString())
+  await page.click('[value="password"]')
+  return email
+}
+
+test.describe("django-ory-cloud", () => {
+  for (const app of [
+    { url: "http://localhost:4000/", name: "django" },
+  ]) {
+    test.describe(app.name, async () => {
+      test("able to use login and sign up", async ({ page }) => {
+        await page.goto(app.url, { waitUntil: "networkidle" })
+
+        await page.locator('text=Sign in').click()
+        await expect(page).toHaveURL(/.*\/ui\/login.*/)
```

Comment:
I haven't tried it but [toContainUrl](https://elaichenkov.github.io/playwright-expect/modules/tocontainurl.html) might be simpler:

```suggestion
        await expect(page).toContainUrl("/ui/login")
```

Reply:
Going after this: https://playwright.dev/docs/test-assertions, i think toContainUrl is from [an extra library](https://elaichenkov.github.io/playwright-expect/). I dont think we use it currently, do you think it would be good to add?

Reply:
Oh right, [playwright-expect](https://elaichenkov.github.io/playwright-expect/index.html) is different from [expect-playwright](https://github.com/playwright-community/expect-playwright). Given that, I'm okay with leaving this out. When in doubt, leave it out!